core/src/main/java/com/klaytn/caver/methods/response/KlayRewards.java

```diff
@@ -93,11 +94,11 @@ public String getProposer() {
         public String getStakers() {
             return this.stakers;
         }
-        public String getKgf() {
-            return this.kgf;
+        public String getKff() {
```

Comment:
minor thing: don't we need to change "kir" or "poc" in GovernanceStakingInfo like this?

Reply:
You are right, thanks. The test passed because It doesn't check every field.